bot/extensions/language.py

```diff
@@ -0,0 +1,73 @@
+from discord.ext.commands import Cog
+from discord import Message
+from logging import info
+from nltk.tokenize import TweetTokenizer
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+import nltk
+
+
+class LinusCog(Cog):
```

Comment:
I'm a believer of the **one file one class** and that both should have the same name. For now it's still correct and we'll change it in the future because it's not that bad. Both names are still well choosen and make sens.